src/main/webapp/app/overview/course-learning-goals/course-learning-goals.component.html

```diff
@@ -1,41 +1,44 @@
-<div class="container" *ngIf="isLoading || (learningGoals && learningGoals.length > 0) || (prerequisites && prerequisites.length > 0); else noData">
-    <div *ngIf="isLoading" class="d-flex justify-content-center">
-        <div class="spinner-border" role="status">
-            <span class="sr-only">{{ 'loading' | artemisTranslate }}</span>
+<div class="row">
+    <div class="col-12 col-lg-8">
+        <div *ngIf="countLearningGoals > 0" class="mt-3 position-relative">
+            <div class="exercise-row-container mb-3">
+                <jhi-learning-goal-card
+                    *ngFor="let learningGoal of learningGoals; index as i; trackBy: identify"
+                    [learningGoal]="learningGoal"
+                    [courseId]="courseId"
+                ></jhi-learning-goal-card>
+            </div>
         </div>
-    </div>
-
-    <div *ngIf="!isLoading">
-        <h2 *ngIf="prerequisites && prerequisites.length > 0" class="text-center my-4">{{ 'artemisApp.learningGoal.prerequisite.title' | artemisTranslate }}</h2>
-
-        <div class="d-flex flex-wrap gap-3 justify-content-evenly">
-            <jhi-learning-goal-card
-                *ngFor="let prerequisite of prerequisites; index as i; trackBy: identify"
-                [learningGoal]="prerequisite"
-                [isPrerequisite]="true"
-                [displayOnly]="true"
-            ></jhi-learning-goal-card>
+        <div *ngIf="countPrerequisites > 0" class="mt-3 position-relative">
+            <div class="exercise-row-container mb-3">
+                <div class="control-label" (click)="isCollapsed = !isCollapsed">
+                    <fa-icon class="pe-3" [icon]="isCollapsed ? faAngleDown : faAngleUp"></fa-icon>
+                    <span>{{ 'artemisApp.learningGoal.prerequisite.title' | artemisTranslate }}: {{ countPrerequisites }}</span>
+                </div>
+                <div *ngIf="!isCollapsed">
+                    <jhi-learning-goal-card
+                        *ngFor="let prerequisite of prerequisites; index as i; trackBy: identify"
+                        [learningGoal]="prerequisite"
+                        [courseId]="courseId"
+                        [isPrerequisite]="true"
+                    ></jhi-learning-goal-card>
+                </div>
+                <div class="collapsed"></div>
```

Comment:
What's the reason for this empty div

Reply:
Adds a border to the bottom (see also course lecture page).